src/Core/Algorithms/ParticleInCell/GravitySimAlgo.h

```diff
@@ -0,0 +1,33 @@
+
+#ifndef CORE_ALGORITHMS_ParticleInCell_GravitySimAlgo_H
+#define CORE_ALGORITHMS_ParticleInCell_GravitySimAlgo_H
+
+#include <Core/Datatypes/Matrix.h>
+#include <Core/Datatypes/DenseMatrix.h>
+#include <Core/Datatypes/DenseColumnMatrix.h>
+
+#include <string>
+#include <sstream>
+#include <vector>
+#include <algorithm>
+
+#include <Core/Algorithms/Base/AlgorithmVariableNames.h>
+#include <Core/Algorithms/Base/AlgorithmBase.h>
+#include <Core/Algorithms/ParticleInCell/share.h>
+
+namespace SCIRun {
+namespace Core {
+namespace Algorithms {
+namespace ParticleInCell {
+
+class SCISHARE GravitySimAlgo : public AlgorithmBase
+    {
+/*
+
+    public:
+      GravitySimAlgo();
```

Comment:
need this line

Reply:
https://github.com/SCIInstitute/SCIRun/blob/master/src/Core/Algorithms/Math/BooleanCompareAlgo.h#L62